.github/bot/README.md

```diff
@@ -0,0 +1,139 @@
+# EIP Linting Bot
+
+This Github Actions integrated bot lints EIPs and provides feedback for authors, its goal is to catch simple problems and merge simple changes automatically.
+# Usage
+
+```yml
+on: [pull_request]
+
+jobs:
+  hello_world_job:
+    runs-on: ubuntu-latest
+    name: EIP Auto-Merge Bot
+    steps:
+      - name: auto-merge-bot
+        uses: ./.github/bot
+        id: auto-merge-bot
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
+```
+
+# Contributing
+## Development Enviornment Setup
+### Requirements
+1) node package manager (npm)
+2) Github Token 
+3) Forked Repo
+4) node version manager (nvm)
```

Comment:
I don't see any references to `nvm` in here, can this requirement be removed (perhaps instead require `nodejs`)?